agent/node.go

```diff
@@ -49,6 +49,10 @@ type NodeConfig struct {
 	// Secret to be used on the first certificate request.
 	Secret string
 
+	// ExternalCAURLs is a list of initial URLs to which a manager node
+	// will make certificate signing requests for node certificates.
+	ExternalCAURLs []string
```

Comment:
What I mean is that these options are mixing those intended for the local node and those destined to become cluster config.

Sorry, this is way out of scope for this PR and swarmkit's mistake.